common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -118,53 +121,60 @@ def get_queryset(self):
         return CourseEntitlement.objects.all().select_related('user').select_related('enrollment_course_run')
 
     def create(self, request, *args, **kwargs):
+        support_details = request.data.pop('support_details', None)
```

Comment:
do you want to default to []?

Reply:
suppose I could since it's also a False-y value, but since I'm only actually treating the object as a list if it's not False-y I figured it wasn't entirely necessary. However, for the sake of type consistency I could use an empty list as the False-y value.

Reply:
That was what I was thinking, additionally I think elsewhere you had implemented it as defaulting to []